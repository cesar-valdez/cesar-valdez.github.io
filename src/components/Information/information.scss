@import '../../scss/variables/colors';

.pi_Card {
	border-radius: 4px;

	&::before {
		content: " ";
		background-image: url(../../img/perfil.jpg);
		background-size: cover;
		background-position: top;
		max-width: 100%;
		width: 100%;
		height: 18em;
		display: inline-block;
	}

	&-information {
		padding: 1em 2em;
		color: $gray-light;

		&-name {
			font-size: 1.7em;
			display: inline-block;
		}

		&-dot {
			width: 4px;
			height: 4px;
			background: $blue;
			display: inline-block;
			border-radius: 50%;
			margin-left: 0.2em;
			vertical-align: unset;
		}

		&-ocupation {
			font-size: 1em;
			margin-bottom: 0.5em;

			&-place {
				color: $green;
			}
		}

		&-description {
			margin-bottom: 1em;
		}
	}
}