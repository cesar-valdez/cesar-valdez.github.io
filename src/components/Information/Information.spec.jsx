import React from 'react';
import { shallow } from 'enzyme';

import Information from './Information';

describe('Information', () => {
    const information = shallow(<Information />);

    it('should render information component', () => {
        expect(information).toMatchSnapshot();
    })
});