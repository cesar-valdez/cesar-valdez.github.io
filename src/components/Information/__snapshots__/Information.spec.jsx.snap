// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Information should render information component 1`] = `
<div>
  <h1
    className="pi_Card-information-name"
  >
    César Valdez
  </h1>
  <span
    className="pi_Card-information-dot"
  />
  <h2
    className="pi_Card-information-ocupation"
  >
    Front-end developer at 
    <span
      className="pi_Card-information-ocupation-place"
    >
      Mifiel
    </span>
  </h2>
  <p
    className="pi_Card-information-description"
  >
    Hi guys! my name is César, I am front-end developer. I have more than 2 years of experience creating web applications under the following tenchologies: HTML, CSS, SASS, Bootstrap, Foundation, Wordpress, jQuery and React, always taking into account a good semantic structure in HTML, therefore responsible for responsive design for all devices and browsers, good user experience and optimization of the applications.
  </p>
</div>
`;
